filebeat/inputsource/common/streaming/listener.go

```diff
@@ -23,8 +23,11 @@ import (
 	"context"
 	"fmt"
 	"net"
+	"strings"
 	"sync"
 
+	"github.com/pkg/errors"
```

Comment:
Tangentially related, github.com/pkg/errors has been archived, does work that it looks like we never use and is arguably harder to reason about than the equivalent offering in stdlib. A little while ago I looked into what would be required to migrate to pure stdlib error handling with the same functionality and 99.9% can be done mechanically over all of beats without trying hard, but the change is on the order of 5,000 lines across 500 files.